docs/features/lambda_versioning.md

```diff
@@ -1,80 +0,0 @@
----
-id: lambda_versioning
-title: Lambda Versioning
-hide_title: true
----
-
-# Lambda Versioning
```

Comment:
Removing doc as feature was removed.

Reply:
I guess with an eye to the fact that we'll re-add the doc when the feature is re-implemented?

Reply:
Yeah.  We can always pull from a prior copy if needed.